gen-apidocs/generators/writer.go

```diff
@@ -62,15 +58,7 @@ func GenerateFiles() {
 		title = "Kubernetes API Reference Docs"
 	}
 
-	var writer DocWriter
-	if *Backend == "brodocs" {
-		writer = NewMarkdownWriter(config, copyright, title)
-	} else if *Backend == "go" {
-		writer = NewHTMLWriter(config, copyright, title)
-	} else {
-		panic(fmt.Sprintf("Unknown backend specified: %s", *Backend))
-	}
-
+    writer := NewHTMLWriter(config, copyright, title)
```

Comment:
nit: indentation of line 61?

Reply:
will check

Reply:
indentation should be one TAB according Go convention. I have got TABs autoconverted to 4 spaces in my VIM. Will fix.